addons/editor-devtools/DevTools.js

```diff
@@ -36,64 +34,12 @@ export default class DevTools {
       this.initInner(root);
     }
   }
-  async addContextMenus() {
-    const blockly = await this.addon.tab.traps.getBlockly();
-    const oldCleanUpFunc = blockly.WorkspaceSvg.prototype.cleanUp;
-    const self = this;
-    blockly.WorkspaceSvg.prototype.cleanUp = function () {
-      if (self.addon.settings.get("enableCleanUpPlus")) {
-        self.doCleanUp();
-      } else {
-        oldCleanUpFunc.call(this);
-      }
-    };
 
-    let originalMsg = blockly.Msg.CLEAN_UP;
-    if (this.addon.settings.get("enableCleanUpPlus")) blockly.Msg.CLEAN_UP = this.m("clean-plus");
-    this.addon.settings.addEventListener("change", () => {
-      if (this.addon.settings.get("enableCleanUpPlus")) blockly.Msg.CLEAN_UP = this.m("clean-plus");
-      else blockly.Msg.CLEAN_UP = originalMsg;
-    });
-
-    this.addon.tab.createBlockContextMenu(
```

Comment:
This context menu item should stay in devtools. It's the one for pasting blocks.